autoscaler/README.md

```diff
@@ -0,0 +1,37 @@
+# Horizontal Self Scaler container
+
+This container image watches over the number of schedulable nodes in the cluster and resizes
+the number of replicas in the required controller.
+
+Usage of pod_nanny:
```

Comment:
Also, there's already a thing called pod_nanny, that manages heapster vertical size.